main.js

```diff
@@ -22,6 +23,16 @@ const logging = require('./app/logging');
 const windowState = require('./app/window_state');
 const { createTemplate } = require('./app/menu');
 
+addUnhandledErrorHandler({
+  logger: (error) => {
+    console.log(
+      'main.js: Uncaught error or unhandled promise rejection:',
+      error && error.stack ? error.stack : error
```

Comment:
This might be a good opportunity to spend the time to remove the current directory from any paths on the call stack, so sensitive information doesn't get into the logs.

Reply:
Good idea. Let me look into that.